qms-tests/sanity/tests/documents/REQ-05.spec.ts

```diff
@@ -51,4 +52,47 @@ test.describe('ISO 13485, 4.2.4 Control of documents', () => {
 
     await attachScreenshot('TESTS-381_workspace_created.png', page)
   })
+
+  test('TESTS-382. Create new a new Effective Template with category External', async ({ page }) => {
+    await allure.description('Requirement\nUsers need to create a new template')
+    await allure.tms('TESTS-382', 'https://tracex.hc.engineering/workbench/platform/tracker/TESTS-382')
+    const leftSideMenuPage = new LeftSideMenuPage(page)
+    const category = faker.word.words(2)
+    const description = faker.lorem.sentence(1)
+    const code = faker.word.words(2)
+    const title = faker.word.words(2)
+
+    await leftSideMenuPage.clickButtonOnTheLeft('Documents')
+    await test.step('2. Create a new category', async () => {
+      const documentContentPage = new DocumentContentPage(page)
+      await documentContentPage.selectControlDocumentSubcategory('Categories')
+      await documentContentPage.clickOnAddCategoryButton()
+      await documentContentPage.fillCategoryForm(category, description, code)
+      await documentContentPage.clickNewDocumentArrow()
+      await documentContentPage.clickNewTemplate()
+      await createTemplateStep(page, title, description, category)
```

Comment:
This statement produces `await test.step('2. Create a new template'` within `await test.step('2. Create a new category'` - doesn't look right?